src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -89,7 +89,7 @@ namespace Platform
     std::string GetInstallPath()
     {
         // 1. Try command line argument
-        auto path = std::string(gCustomOpenRCT2DataPath);
+        auto path = gCustomOpenRCT2DataPath;
```

Comment:
```suggestion
        const auto& path = gCustomOpenRCT2DataPath;
```
`const &` to avoid a copy here?

Reply:
Now that changing to `std::string` is not necessary anymore, this entire variable can be ditched and `gCustomOpenRCT2DataPath` can be passed directly instead.